jupyterhub_singleuser_profiles/images.py

```diff
@@ -27,6 +27,16 @@ def get_default(self):
 
         return ''
 
+    def get_info(self, image_name):
+        imagestream_list = self.openshift.get_imagestreams(IMAGE_LABEL+'=true')
+
+        if len(imagestream_list.items) == 0:
```

Comment:
You don't need this check, if the list has lenght 0, the for loop will never do any work..